packages/crayons-extended/custom-objects/src/components.d.ts

```diff
@@ -368,14 +507,15 @@ export namespace Components {
           * link to show dependent field document
          */
         "dependentFieldLink": string;
+        "dynamicSectionsBetaEnabled": boolean;
         /**
           * svg image to be shown for empty record
          */
         "emptySearchImage": any;
         /**
           * Prop to store the expanded field index
          */
-        "expandedFieldIndex": number;
+        "expandedFieldIndex": {};
```

Comment:
Change variable name to signify what it is, since FieldIndex, specifies that it is singular